pytensor/tensor/conv/abstract_conv.py

```diff
@@ -633,7 +633,7 @@ def border_mode_to_pad(mode, convdim, kshp):
             ):
                 if len(m) != 2:
                     raise NotImplementedError(
-                        f"Asymmetric padding not implemented for {len(m)}d"
+                        f"Asymmetric padding not implemented for {len(m)}"
```

Comment:
Was the `d` referring to dimensions? As in `m dimensions`?

Reply:
Ah yes indeed, great catch again. I was mixed up with percent formatting where `d` stands for `double`.